From c74992c62a32080208baf137d61f7d73b0cade2d Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Tue, 16 Feb 2021 16:57:36 +0100
Subject: [PATCH] system76-launch plugin requires gusb
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Otherwise the following build error happens:

  ../fwupd-1.5.6/plugins/system76-launch/fu-system76-launch-device.c:117:9: error: ‘G_USB_DEVICE_CLAIM_INTERFACE_BIND_KERNEL_DRIVER’ undeclared (first use in this function)
    117 |         G_USB_DEVICE_CLAIM_INTERFACE_BIND_KERNEL_DRIVER,
        |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  ../fwupd-1.5.6/plugins/system76-launch/fu-system76-launch-device.c:133:11: error: ‘G_USB_DEVICE_CLAIM_INTERFACE_BIND_KERNEL_DRIVER’ undeclared (first use in this function)
    133 |           G_USB_DEVICE_CLAIM_INTERFACE_BIND_KERNEL_DRIVER,
        |           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
---
 plugins/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/meson.build b/plugins/meson.build
index 138c47a4..12c7cb31 100644
--- a/plugins/meson.build
+++ b/plugins/meson.build
@@ -25,7 +25,6 @@ subdir('ata')
 subdir('elantp')
 subdir('optionrom')
 subdir('superio')
-subdir('system76-launch')
 subdir('thelio-io')
 subdir('wacom-raw')
 endif
@@ -59,6 +58,7 @@ subdir('solokey')
 subdir('steelseries')
 subdir('synaptics-cxaudio')
 subdir('synaptics-prometheus')
+subdir('system76-launch')
 subdir('vli')
 subdir('wacom-usb')
 
-- 
2.30.1

